pattern_import_export/models/ir_exports_line.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, exceptions, fields, models
+
+
+class IrExportsLine(models.Model):
+    _inherit = "ir.exports.line"
+
+    select_tab_id = fields.Many2one("ir.exports.select.tab", string="Select tab")
+    is_many2x = fields.Boolean(
```

Comment:
maybe it will be smatter to have is_many2one and _is_many2many and share the same compute method?

Reply:
I don't like sharing compute method. Because if you forget a case, it could put an invalid value into another field computed by this same function. But if you really want this modification, I can do it into the #9 because I do it again.